geomstats/_backend/pytorch/__init__.py

```diff
@@ -324,7 +331,17 @@ def einsum(*args, **kwargs):
         input_str = input_str_list[0] + ',' + input_str_list[1]
         einsum_str = input_str + '->' + output_str
 
-        return torch.einsum(einsum_str, tensor_a, tensor_b, **kwargs)
+        result = torch.einsum(einsum_str, tensor_a, tensor_b, **kwargs)
```

Comment:
Was the einsum_str modified as compared to the input one?

Reply:
yes, in the cases in if/else above?